src/content/app-platform/getting-started/index.md

```diff
@@ -0,0 +1,265 @@
+---
+linkTitle: Getting started
+title: Getting started with App Platform
+description: Guide to deploying apps using kubectl gs and the Giant Swarm Management API.
+weight: 20
+menu:
+  main:
+    parent: app-platform
+last_review_date: 2021-07-07
+owner:
+  - https://github.com/orgs/giantswarm/teams/team-batman
+user_questions:
+  - How can I deploy an app using an App CR?
+  - How can I configure an App CR?
+  - How can I see which apps are available using the Management API?
+---
+
+# Getting started with App Platform
+
+## Overview
+
+The _Giant Swarm App Platform_ is built on top of [Helm](https://helm.sh/) and allows you to manage apps and their configurations represented by App Custom Resources (CRs) for multiple clusters, from a single place: the [management cluster]({{< relref "/general/management-clusters" >}}).
+
+In this guide, we will install the NGINX Ingress Controller app. We will do this by using kubectl, to create
+an [App]({{< relref "/ui-api/management-api/crd/apps.application.giantswarm.io.md" >}}) CR using the Kubernetes API of your management cluster.
+
+App CRs can be created this way via your automation or our Web UI (See: [guide]({{< relref "/getting-started/ingress-controller" >}})).
+
+In general, you can manage App CRs with any tool that can communicate with the Kubernetes API such as Helm or GitOps tools (like Argo CD or Flux CD).
+
+## Setting up
+
+You can access your management cluster using the [kubectl gs login]({{< relref "/ui-api/kubectl-gs/login" >}})
+command of our kubectl plugin. See [here]({{< relref "/ui-api/kubectl-gs/installation" >}})
+for how to install it. If you already have it installed please upgrade to the
+latest version.
+
+In the management cluster your App CRs are stored in a namespace with the same
```

Comment:
The namespace is named after the cluster id right?

I am no native speaker, so take this with a grain of salt :salt:

```suggestion
App CRs are stored on the management cluster in the namespace named after the ID of your workload cluster.
```

Reply:
oh ignore me, I should've read the whole sentence before commenting :facepalm: